R/plot_variation.r

```diff
@@ -35,8 +35,10 @@ plot_variation <- function(input_data = NULL, xlab = "Area", ylab = "Rate", type
       ggplot2::ggplot(data = input_data,
       ggplot2::aes(x = get("area_name"), y = get("value"), fill = get("brks"),
                    text = paste("<b>", get("area_name"), "</b><br>",
-                                get("type"), ":", format(get("value"), decimal.mark = decimal_mark, digits = 2), "<br>",
-                                get("numerator_name"), ":", format(get("numerator"), big.mark = big_mark)))) +
+                                get("type"), ":", format(get("value"),
+                                big.mark = big_mark, decimal.mark = decimal_mark, digits = 2), "<br>",
+                                get("numerator_name"), ":", format(get("numerator"),
+                                big.mark = big_mark, decimal.mark = decimal_mark, digits = 2)))) +
```

Comment:
Maybe use `paste0` instead of `paste` and replace `":"` with `": "`, to avoid spaces around all elements?

Reply:
sure!  Changes made.  :-)  But what do you mean by to avoid spaces around all elements?

Reply:
All elements you send in to `paste()` will be separated by a space. So `paste(get("type"), ":", format(get("value"))` will turn into the string `"Rate : 1234"`.

`paste0(get("type"), ":", format(get("value"))` and `paste(get("type"), ":", format(get("value"), sep = "")` will turn into `"Rate:1234"`.

`paste0(get("type"), ": ", format(get("value"))` will turn into `"Rate: 1234"`, which I guess is what we want.